[change] Exclude staff users from delete_unverified_users #353 (#354)
openwisp/openwisp-radius: docs/source/user/management_commands.rst

more than specified duration and have no associated radius session.
This feature is needed to delete users who have registered but never
completed the verification process.

> Staff users will be excluded from the deletion process.

Reviewer: ```suggestion
**Staff users will not be deleted by this management command.**
```

> **Staff users will not be deleted from this management command.**

Reviewer: ```suggestion
**Staff users will not be deleted by this management command.**
```
